src/main/java/com/netgrif/application/engine/configuration/locale/CustomLocaleResolver.java

```diff
@@ -0,0 +1,46 @@
+package com.netgrif.application.engine.configuration.locale;
+
+import org.springframework.context.i18n.LocaleContext;
+import org.springframework.web.servlet.LocaleContextResolver;
+import org.springframework.web.servlet.i18n.AcceptHeaderLocaleResolver;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Locale;
+
+/**
+ * A custom implementation of locale resolver to store the time zone offset in
+ * request context
+ * */
+public class CustomLocaleResolver implements LocaleContextResolver {
```

Comment:
No need for "Custom" prefix, all other related classes are named just "Locale..." so this one should be too

Reply:
Done